src/styles/helpers/_scrollbar.scss

```diff
@@ -0,0 +1,30 @@
+@mixin fd-themed-scrollbar() {
```

Comment:
Please move this mixin in the `scrollbar.scss` file. Let's see in future if it will be reused, then we can move it to a separate file, but for now there's no value of keeping it in a separate file.